backend/server.js

```diff
@@ -2,47 +2,48 @@
 // where your node app starts
 
 // init project
-const express = require("express");
-const path = require("path");
+const express = require('express');
```

Comment:
I was wondering about the double quotes. I did the same as I didn't want to not conform but I'm glad we switched to singles.